### PR TITLE
HW19_Georgy_Neizvestny

### PR DESCRIPTION
**Pull request by Egorius96**

### CODE REVIEW

File: MyHW/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
```
Comment (tamkovich) [inline MyHW/admin.py]: please remove empty files

File: MyHW/models.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
+
+# Create your models here.
```
Comment (tamkovich) [inline MyHW/models.py]: please remove empty files

File: MyHW/tests.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (tamkovich) [inline MyHW/tests.py]: please remove empty files

File: MyHW/views.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+def hw1901(request):
+    messages = {request.POST.get('message01', ''):len(request.POST.get('message01', '')),
+                request.POST.get('message02', ''):len(request.POST.get('message02', '')),
+                request.POST.get('message03', ''):len(request.POST.get('message03', ''))}
+    keymax = max(messages, key=messages.get)
+    return render(request, 'HW_1901.html', context={'result':keymax})
+
+def hw1902(request):
+    date = request.POST.get('date', '')
```
Comment (tamkovich) [inline MyHW/views.py]: please use datetime.strftime function to parse dates

File: MyHW/views.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        date = f'С новым {date[0:4]} годом!'
+    return render(request, 'HW_1902.html', context={'date':date})
+
+def additionaltask1(request):
+    req = request.POST.get('username', 'World')
+    if req == '':
+        req = 'World'
+    return render(request,
+                  'additional_task1.html',
+                  context={"username": f'{req}'})
```
Comment (tamkovich) [inline MyHW/views.py]:
```suggestion
                  context={"username": req})
```

File: MyHW/admin.py (hunk 1 above)
Comment (Egorius96) [reply MyHW/admin.py]: done

File: MyHW/models.py (hunk 2 above)
Comment (Egorius96) [reply MyHW/models.py]: done

File: MyHW/tests.py (hunk 3 above)
Comment (Egorius96) [reply MyHW/tests.py]: done

File: MyHW/views.py (hunk 4 above)
Comment (Egorius96) [reply MyHW/views.py]: done

**Pull request closed without merging**